test/_prepare_outputs.py

```diff
@@ -1,6 +1,7 @@
 import glob
 import os
 import subprocess
+import sys
```

Comment:
This is not required if you have a venv.

Reply:
Does it hurt though? If not, it seems to me better to keep it.

Reply:
Yes.  It hurts.  Please read `Why do virtual environments exist?` from a long-term core Python developer and head of Python development for VSCode...  https://snarky.ca/how-virtual-environments-work

Reply:
> In the end, whether you type `python` after activation or `.venv/bin/python` makes no difference to Python

Hum I don't know if I'm reading that right, but it seems to me this implies it would not hurt, as it says it makes no difference, no? 🤔

Reply:
It says that the goal of using a virtualenv to avoid installing your dependencies into your system Python.
Doing so can cause dependency conflicts that can become unresolvable when multiple projects are installed.

Reply:
Yes, I do see the upsides of using a virtualenv, much less troublesome than spawning a docker container. But I thought you were saying “it hurts to use `sys.executable` instead of `"python"`”, and – I guess this where I’m missing something – I don’t see how that relates to virtualenv or how it would hurt.